core/src/main/java/org/mskcc/cbio/portal/util/GeneticProfileReader.java

```diff
@@ -75,8 +75,14 @@ public static GeneticProfile loadGeneticProfile(File file) throws IOException, D
       if (existingGeneticProfile != null) {
          // the dbms already contains a GeneticProfile with the file's stable_id. This scenario is not supported
     	 // anymore, so throw error telling user to remove existing profile first:
-         throw new RuntimeException("Error: genetic_profile record found with same Stable ID as the one used in your data:  "
+         if (!existingGeneticProfile.getDatatype().equals("MAF"))
+         {
+             throw new RuntimeException("Error: genetic_profile record found with same Stable ID as the one used in your data:  "
                   + existingGeneticProfile.getStableId() + ". Remove the existing genetic_profile record first.");
+         }
+         else {
+             return existingGeneticProfile;
```

Comment:
just a FYI - this should be fine since the `mutation` table has an active constraint (I introduced in `migration.sql` some time ago - see `UQ_MUTATION_EVENT_ID_GENETIC_PROFILE_ID_SAMPLE_ID` ) to prevent duplicated data that could occur by mistake (one of the possible mistakes with this new code would be to submit the same mutation data file twice - if this mistake is made, MysqlBulkLoader should throw an exception because of the UQ constraint above).